### DISCUSSION TITLE
Directory Structure Not Working/Understandable

### DISCUSSION THREAD

**Discussion by khanfarhan10** [mmistakes/minimal-mistakes#2703] [Q&A] [answered]
I'm sure that this question has an easy answer, but I absolutely cannot understand the navigation structure.
I was able to get [these pages](https://drive.google.com/drive/folders/1P7SC7izI9biWiKxQbZv9uOqb7Idncu5y?usp=sharing)
 up and running with all my tabs placed modifying the navigation.yml in the _data directory. 
But the problem is that when I open any of the tabs (say projects, posts or courses ) I am only able to view the markdown files under _posts and not under _courses or _projects. So how do I achieve this as all of my tabs are pointing to _posts.

# What I am trying to achieve
Projects should show text about recursion under the _projects folder.
Posts should show text about (Something : Might Run) under the _posts folder. (this works fine for now)
Courses should show text about Python course completion under the _courses folder.

I'm running my website locally on ruby but have uploaded it to github pages now : https://khanfarhan10.github.io/ . Thanks in advance, much appreciated. I'm new to Jekyll and ruby.

**Comment by khanfarhan10** [comment]
Now that I have a basic understanding of jekyll, I still don't understand how the posts, research and project pages are generated on my github pages website : [khanfarhan10.github.io](https://khanfarhan10.github.io/).

I want to know how to separately add pages for each of them, viz , posts research and projects. The source code is [at my repository](https://github.com/khanfarhan10/khanfarhan10.github.io)

**Comment by iBug** [comment, marked as answer]
Check the docs: <https://jekyllrb.com/docs/collections/>

You need to explicitly enable collections and render them into pages in your `_config.yml`:

```yaml
collections:
  my_collection:
    output: true
  my_another_collection:
    output: true
```
